scripts/debug_tools/prepare_analyzer_cmd.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+import argparse
+import re
+import failure_lib as lib
```

Comment:
This is not a system-level import, so let's format it accordingly, and below all system imports.